db_stress_tool/db_stress_test_base.cc

```diff
@@ -2563,6 +2563,10 @@ void StressTest::Open() {
             s = static_cast_with_check<DBImpl>(db_->GetRootDB())
                     ->TEST_WaitForCompact(true);
             if (!s.ok()) {
+              for (auto cf : column_families_) {
+                delete cf;
+              }
+              column_families_.clear();
               delete db_;
```

Comment:
Unless we're in a destructor or otherwise the "last place" a `delete db_` might happen, I prefer to set `db_ = nullptr` to make diagnosis of double-delete easy.